#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_qmake_use_qt4_tools.dpatch by Brian Nelson <pyro@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Ensures qmake generates Makefiles that use the -qt4 tools and
## DP: that use recursively link against all indirectly-used libraries.

@DPATCH@
diff -urNad qt4-x11-4.2.1~/mkspecs/common/linux.conf qt4-x11-4.2.1/mkspecs/common/linux.conf
--- qt4-x11-4.2.1~/mkspecs/common/linux.conf	2006-10-20 11:35:09.000000000 -0400
+++ qt4-x11-4.2.1/mkspecs/common/linux.conf	2006-10-30 22:24:57.000000000 -0500
@@ -19,8 +19,8 @@
 QMAKE_LIBS_OPENGL_QT  = -lGL
 QMAKE_LIBS_THREAD     = -lpthread
 
-QMAKE_MOC             = $$[QT_INSTALL_BINS]/moc
-QMAKE_UIC             = $$[QT_INSTALL_BINS]/uic
+QMAKE_MOC             = $$[QT_INSTALL_BINS]/moc-qt4
+QMAKE_UIC             = $$[QT_INSTALL_BINS]/uic-qt4
 
 QMAKE_AR              = ar cqs
 QMAKE_RANLIB          =
diff -urNad qt4-x11-4.2.1~/mkspecs/linux-g++/qmake.conf qt4-x11-4.2.1/mkspecs/linux-g++/qmake.conf
--- qt4-x11-4.2.1~/mkspecs/linux-g++/qmake.conf	2006-10-20 11:35:55.000000000 -0400
+++ qt4-x11-4.2.1/mkspecs/linux-g++/qmake.conf	2006-10-30 22:25:28.000000000 -0500
@@ -4,7 +4,7 @@
 
 MAKEFILE_GENERATOR	= UNIX
 TEMPLATE		= app
-CONFIG			+= qt warn_on release incremental link_prl
+CONFIG			+= qt warn_on release incremental
 QT			+= core gui
 QMAKE_INCREMENTAL_STYLE = sublib
 
